providers/k8s/connection/manifest/connection.go

```diff
@@ -81,12 +81,6 @@ func NewConnection(id uint32, asset *inventory.Asset, opts ...Option) (shared.Co
 		asset.Name = clusterName
 	}
 
-	platformId, err := c.AssetId()
```

Comment:
We already do this during discovery:
https://github.com/mondoohq/cnquery/blob/main/providers/k8s/resources/discovery.go#L118-L128